### ISSUE TITLE
Questions on Experimental Design Choices

### ISSUE THREAD

**Issue by wenzhuwusong** [bw-park/ACSSM#1]
Amazing work! I have four questions about your methodology:

1. I could not locate the detailed hyperparameter selection process. Could you elaborate on the specific strategies employed for determining the optimal hyperparameters? 
2. Why do you use a fixed number of training epochs, rather than a model selection strategy?
3. I found that in the USHCN and PhysioNet experiments, the validation sets were merged with training data for final model training. Could you clarify the original purpose of the validation partition ?
4. What was the reasoning behind reporting evaluation metrics on normalized data, instead of transforming them back to their original units for more direct interpretability?

Thank you for your time and insights!
